src/main/java/com/harena/api/endpoint/rest/controller/PatrimoineController.java

```diff
@@ -24,4 +25,11 @@ public GetPatrimoines200Response getPatrimoines(
         service.getPaginatedPatrimoines(page, pageSize).stream().map(mapper::toRest).toList();
     return new GetPatrimoines200Response().data(data);
   }
+
+  @GetMapping("/patrimoines/{nom_patrimoine}")
+  public GetPatrimoines200Response getPatrimoineByName(
```

Comment:
The return type of this method should be `Patrimoine`
`GetPatrimoines200Response` is only used for methods that return an array of `Patrimoine`